### PR TITLE
Accessibility updates

### PR DESCRIPTION
**Pull request by mishmanners**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [x] The commit message follows the [conventional commits][cc] guidelines.
- [ ] Tests for the changes have been added (for bug fixes / features).
- [ ] Docs have been added/updated (for bug fixes / features).

### Description

Add accessibility features based on: https://github.blog/2023-10-26-5-tips-for-making-your-github-profile-page-accessible/

Having links next to the text is ideal for accessibility. You have some double links though so I couldn't change those.

### How

NA

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
-[![License][license]](COPYING)
+![MIT license badge](https://img.shields.io/github/license/JanDeDobbeleer/oh-my-posh.svg)
 
-![Release Status][release-status]
+![Build Status badge](https://img.shields.io/github/actions/workflow/status/jandedobbeleer/oh-my-posh/release.yml?branch=main)
 
-[![Release][release-badge]][release]
-[![Documentation][docs-badge]][docs]
+![Release version number badge][release-badge]][release]
+![Documentation link badgeL ohmyposh.dev][docs-badge]][docs]
```
Comment (JanDeDobbeleer) [inline README.md]:
```suggestion
![Documentation link badge ohmyposh.dev][docs-badge]][docs]
```

small typo I believe

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

**Comment by JanDeDobbeleer** [comment]
You'll also need to adjust the commit message to respect the conventional commit guidelines, or, I squash the changes and change the message myself.

**Comment by mishmanners** [comment]
Hmmm, how do I change a commit message?

**Comment by JanDeDobbeleer** [comment]
> Hmmm, how do I change a commit message?

From the web UI I don't know (assuming that's how you did this due to seeing the `patch-1` branch name). I have a guide on that using the CLI and GitKraken [here](https://ohmyposh.dev/docs/contributing/git#i-didnt-stick-to-the-conventional-commit-guidelines). Let me know if that works for you, otherwise I'm going to squash this right here.

**Comment by mishmanners** [comment]
okay that should be good now @JanDeDobbeleer. Let me know if it's all good to go.

**Comment by JanDeDobbeleer** [comment]
> okay that should be good now @JanDeDobbeleer. Let me know if it's all good to go.

@mishmanners you added a third commit, but the initial two still remained. This can be solved using an interactive rebase locally to "squash" all commits in 1, just like the UI here does but with the added benefit of running the checks again to make sure our commit message is compliant. This is necessary to enable automated changelog generation, that way I don't have any work after merging. I solved this for you now, if you want more information on what/how, simply drop a message. Thanks for your contribution!

**Comment by JanDeDobbeleer** [comment]
@all-contributors please add @mishmanners for doc

This is pure magic 🪄! @holopin-bot @mishmanners wizard

**Pull request merged**